src/app/products/products-routing.module.ts

```diff
@@ -256,7 +271,7 @@ const routes: Routes = [
           path: 'products-mix',
           component: ProductsMixComponent,
           resolve: {
-                products: ProductsMixResolver
+            products: ProductsMixResolver
```

Comment:
Remove all these corrections to indentation from this PR. This should not be a part of this.